dev/tests/integration/testsuite/Magento/Framework/Api/SearchCriteria/CollectionProcessor/JoinProcessorTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Framework\Api\SearchCriteria\CollectionProcessor;
+
+use Magento\Framework\Api\SearchCriteria\CollectionProcessor\JoinProcessor\CustomJoinInterface;
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Framework\Data\Collection\AbstractDb;
+use PHPUnit\Framework\TestCase;
+
+class JoinProcessorTest extends TestCase
+{
+    /**
+     * @var \Magento\Framework\ObjectManagerInterface
```

Comment:
This qualifier can be replaced with an import. Import the ObjectManagerInterface and use it as below:
```suggestion
     * @var ObjectManagerInterface
```